File: leetcode/1easy/이경호/2733.cpp
#include<algorithm>
#include<vector>

using namespace std;

class Solution {
public:
  int findNonMinOrMax(vector<int>& nums) {
    int maxElement = *max_element(nums.begin(), nums.end());
    int minElement = *min_element(nums.begin(), nums.end());
    for (int i = 0; i < nums.size(); i++) {
      if (nums[i] != maxElement && nums[i] != minElement) {
        return nums[i];
      }
    }
    return -1;
  }
};

File: leetcode/1easy/오하늘/2733.py
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution:
=======
class Leetcode.이재훈.Solution:
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    def findNonMinOrMax(self, nums: List[int]) -> int:
        nums.sort()
        nums.pop()
        if len(nums) == 0: return -1
        nums.pop(0)
        if len(nums) == 0: return -1
        return nums[0]
